src/Notifier.ts

```diff
@@ -383,6 +383,10 @@ export const Notifier = {
                 // don't bother notifying as user was recently active in this room
                 return;
             }
+            if (SettingsStore.getValue("doNotDisturb")) {
```

Comment:
this should probably also check the feature is enabled to not lock a user into dnd

Reply:
the only way to get access to it though is through the feature flag though? If someone goes through devtools and changes it, that's kinda their fault (the devtools don't prevent someone from making mistakes).

Reply:
I mean if they end up disabling the lab this option should be implicitly disabled as it'll be very unclear why they are missing their notifications

Reply:
I suppose, though overall the labs feature system isn't entirely meant to have safety nets given we don't usually want people to be playing around in there...

Have opened https://github.com/vector-im/element-web/issues/17000 in any case